.github/workflows/integrate.yaml

```diff
@@ -14,16 +16,16 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Install dependencies
-      run: sudo pip3 install black flake8
+      run: sudo pip3 install black flake8 flake8-copyright<0.3
 
     - name: Check black
-      run: black --check .
+      run: black --check --extend-exclude charms/kfp-profile-controller/files/upstream/* .
 
     - name: Check flake8
-      run: flake8
+      run: flake8 --extend-exclude charms/kfp-profile-controller/files/upstream/*
```

Comment:
Is this not covered with the lint `tox` environment?

Reply:
We haven't yet moved the kfp-operators to `pytest-operator` and `tox`.